site/gatsby-site/playwright/probes/probes.ts

```diff
@@ -0,0 +1,40 @@
+import { test } from '../utils';
+import { testUrls } from '../e2e/pages-tests';
+import config from '../../config';
+
+// This array defines the probe tests that are run
+// Each test has a string description, string baseUrl value, and array of string paths
+// that are sent to testUrls(baseUrl,paths)
+[
+	// prod urls
+	// baseUrl: null means use value of config.gatsby.siteUrl
+	{ 
+		title: 'Probing prod home page', 
+		// baseUrl: null means use value of config.gatsby.siteUrl
+		baseUrl: null, 
+		// home page
+		paths: ['/']
+	},
+	{ 
+		title: 'Probing prod submitted page', 
+		// baseUrl: null means use value of config.gatsby.siteUrl
+		baseUrl: null, 
+		// submitted
+		paths: ['/apps/submitted/']
+	},
+	{ 
+		title: 'Probing prod newsdigest page', 
+		// baseUrl: null means use value of config.gatsby.siteUrl
+		baseUrl: null, 
+		// newsdigest
+		paths: ['/apps/newsdigest/']
+	},
+	{ 
+		title: 'Probing staging checklist page', 
+		// baseUrl: null means use value of config.gatsby.siteUrl
+		baseUrl: 'https://staging-aiid.netlify.app', 
+		paths: ['/apps/checklist/']
+	}
+].forEach(({title, baseUrl, paths}) => {
+	test(title, () => testUrls(baseUrl, paths));
```

Comment:
Just checking: I believe `testUrls()` will run nested Playwright tests. What does the output of this test look like?

Reply:
wrt testUrls, I was assuming that this import would override

import { testUrls } from '../e2e/pages-tests';

But my scoping understanding might be wrong.

What does the output of this test look like?

I haven't been able to run this test locally (although I did test/run for syntax locally without imports.   I had some problems with some config/setup/dependency, etc.    I admit I was hoping someone else could run it after fussing with playwright and existing deps/tests for too long.   I'll return to fussing perhaps today, but in following the existing instructions I couldn't get a functional environment to run this test only with something like (paraphrasing):  npx playwright probes/probes.ts...which seems correct via playwright docs.   Again, I'll fuss with it some more and open an issue if update to latest (on staging branch?) doesn't help.   Thanks.